package.json

```diff
@@ -29,7 +29,7 @@
     "posttest": "npm run lint"
   },
   "engines": {
-    "node": ">= 0.6"
+    "node": ">= 0.6 <= 6"
```

Comment:
@kjdelisle any pr checks into 2.x has to drop node 7 to get juggler tests pass, I put it in a separate commit to replace https://github.com/strongloop/loopback-datasource-juggler/pull/1391(discussion also under this PR)

Why that PR is closed:
A few days ago I thought https://github.com/strongloop/loopback-datasource-juggler/pull/1342 would check in soon, and since it also contains `"node": ">= 0.6 <= 6"`, I closed https://github.com/strongloop/loopback-datasource-juggler/pull/1391 so it doesn't need to rebase.